src/Auditor.jl

```diff
@@ -288,6 +288,9 @@ function check_dynamic_linkage(oh, prefix, bin_files;
                     else
                         if !silent
                             @warn("Linked library $(libname) could not be resolved and could not be auto-mapped")
+                            if is_troublesome(libname, platform)
+                                @warn("Depending on $(libname) is known to cause problems at runtime, this needs to be fixed")
```

Comment:
Something scarier like `@error`?  This is very likely going to be an issue.

Reply:
We very clearly need more fine-grained error handling, but `@warn` is fine for now.

Let's call this `is_troublesome_library_link()` to make it a little more explicit what is troublesome.

Reply:
Ok, I also made the warning message more clear about how to fix the issue.